pages/linux/ldapsearch.md

```diff
@@ -0,0 +1,35 @@
+# ldapsearch
+
+> CLI utility for querying an LDAP directory.
+
+- Query LDAP for all items that match filter "{{memberOf}}={{group1}}" and return {{displayName}}:
```

Comment:
Perhaps this would read better?

```suggestion
- Query an LDAP server for all items that are a member of the given group and return the display name:
```

Reply:
Would "return the object's `display name` value", fit? I feel like the user should be nudged to not think of object values as universal.

Reply:
Sounds good to me!

Reply:
After much incremental editing, I think I have this sorted.